quickstarts/synthetics-optimization/config.yml

```diff
@@ -0,0 +1,47 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: synthetics-optimization
+
+# Displayed in the UI (required)
+title: Synthetics Optimization
+
+# Long-form description of the quickstart (required)
+description: |
+  The Synthtics Optimization quickstart aims to help you understand whether you are over your Synthetics usage limits, and if so, how can you identify the right monitors and checks to optimise.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Optimize Synthetics Monitors in your estate to only use what is necessary
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Andrew Newhouse (New Relic)  
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - synthetics
+  - optimization
+  - usage
+  - ping
+  - scripted browser
+  - api test
+  - monitor
+  - availability
+  - simple browser
+
+# Documentation references
+documentation:
+  - name: Synthetics Usage Queries
+    url: >-
+      https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts/#synthetics-usage
+    description: |
+      This document provides additional queries that can be run to determine Synthetics usage.
+
+# Content / Design
+icon: logo.png
+website: https://www.newrelic.com
```

Comment:
```suggestion
website: https://www.newrelic.com

installPlans:
  - third-party-synthetics-endpoint-availability
```

Reply:
Good catch @aswanson-nr !  I wasn't sure about that option when creating the config, but that makes sense to me now!  Thanks for the suggestion and finding that!!